cmd/eno-reconciler/main.go

```diff
@@ -90,16 +88,6 @@ func run() error {
 		return fmt.Errorf("constructing manager: %w", err)
 	}
 
-	err = aggregation.NewSliceController(mgr)
```

Comment:
So the reconciler is now only in charge syncing slices into real resources?

Reply:
Exactly, probably more intuitive anyway